libd-llvm/src/d/llvm/expression.d

```diff
@@ -468,6 +468,7 @@ struct ExpressionGen {
 			auto thisPtr = LLVMBuildExtractValue(builder, dg, m.hasContext, "");
 			auto vtblPtr = LLVMBuildStructGEP(builder, thisPtr, 0, "");
 			auto vtbl = LLVMBuildLoad(builder, vtblPtr, "vtbl");
```

Comment:
and it is. 
because it will only considered to be invariant when they are in the same invariant group.
anyhow I am not going to invest time in this issue anymore.